ocs_ci/helpers/dr_helpers.py

```diff
@@ -1691,14 +1700,18 @@ def replace_cluster(workload, primary_cluster_name, secondary_cluster_name):
     run_cmd(cmd=f"oc delete drcluster {primary_cluster_name} --wait=false")
 
     # Disable DR on hub for each app
-    disable_dr_from_app(secondary_cluster_name)
+    disable_dr_from_app(secondary_cluster_name, rdr=True)
     logger.info("DR configuration is successfully disabled on each app")
 
     # Remove DR configuration from hub and surviving cluster
     logger.info("Running Remove DR configuration script..")
     run_cmd(cmd=f"chmod +x {constants.REMOVE_DR_EACH_MANAGED_CLUSTER}")
     run_cmd(cmd=f"sh {constants.REMOVE_DR_EACH_MANAGED_CLUSTER}")
 
+    time.sleep(60)
```

Comment:
what's the reason for waiting? Would it be possible to replace it with a TimeoutSampler?